src/qibolab/platforms/abstract.py

```diff
@@ -105,9 +106,9 @@ def __init__(self, name, runcard):
         self.relaxation_time = None
         self.sampling_rate = None
 
-        self.native_single_qubit_gates = {}
-        self.native_two_qubit_gates = {}
-        self.two_qubit_natives = set()
+        self.single_qubit_natives = {}
+        # TODO: find a better initialization to start with an empty flag class
+        self.two_qubit_natives = TwoQubitNatives.CZ and TwoQubitNatives.iSWAP
```

Comment:
@AleCandido @stavros11 You know how i can initialize two_qubit_natives as a empty TwoQubitNatives flag class?
this should work but i think there is a better way

Reply:
Since they are bit masks, I believe it should be just:
```suggestion
        self.two_qubit_natives = TwoQubitNatives(0)
```

Check the example in the docs for `Flag.__bool__`
https://docs.python.org/3/library/enum.html#enum.Flag

Reply:
just found it, thanks